client/src/pages/Home.js

```diff
@@ -1,30 +1,78 @@
 import MainTable from "../Component/MainTable";
 import SMSModal from "../Component/SMSModal";
 import "./Home.css";
-import { useState } from "react";
-import UploadFile from "./UploadFile";
+import React, { useState, useEffect } from "react";
+import Alert from "react-bootstrap/Alert";
+
+
 
 export function Home() {
-	const [checkedCheckboxes, setCheckedCheckboxes] = useState([]);
+	const [selectedClients, setSelectedClients] = useState([]);
+	const [clientData, setClientData] = useState([]);
+	const [error, setError] = useState(null);
+	const [isLoading, setIsLoading] = useState(false);
+	const [success, setSuccess] = useState(false);
```

Comment:
```suggestion
	const [isSuccess, setIsSuccess] = useState(false);
```
Minor: Some teams like the convention of starting booleans with `is` or `has`, which we could follow here as we're also using it for `isLoading` above